test/acceptance/cli-test.js

```diff
@@ -195,7 +195,7 @@ describe('ember-template-lint executable', function () {
         let result = await run(
           ['--filename', 'app/templates/application.hbs', '<', 'app/templates/application.hbs'],
           {
-            shell: false,
+            shell: true,
```

Comment:
FWIW, I don't think this specific test _should_ work on Windows. The `<` style redirection is pretty shell dependent AFAIK.

Reply:
Correct! I finally understood how to properly make those tests

Reply:
> The < style redirection is pretty shell dependent AFAIK.

🤦, I missed that.

But I was somehow lucky because `<` style redirection is not supported by PowerShell on Windows. But - and I didn't know it - Windows command prompt seems to support it.

I've not been able to make execa work with PowerShell but execa works by default with cmd.exe on Windows (namely Windows command prompt).

So we could activate that option in the test suite. I gave it a try in https://github.com/ember-template-lint/ember-template-lint/pull/1298